cmd/commands/runtime.go

```diff
@@ -783,6 +785,16 @@ func preInstallationChecks(ctx context.Context, opts *RuntimeInstallOptions) err
 		return fmt.Errorf("existing runtime check failed: %w", err)
 	}
 
+	if !opts.SkipClusterChecks {
+		err = util.RunNetworkTest(ctx, opts.KubeFactory, cfConfig.GetCurrentContext().URL)
+		handleCliStep(reporter.InstallStepRunPreCheckNetworkTest, "Running network tests", err, false)
```

Comment:
* Please change the step name to `InstallStepRunPreCheckClusterChecks`
* Please move the reporting outside the if statement. (We should never report conditionally)